pebble-tool attempts to call a Python script here, by passing the script to the
Python executable. Nix wraps Python scripts in shell scripts, so that necessary
Nix-specific environment variables can be set. Calling that shell script with
the Python executable is going to fail, for obvious reasons. We can fix this by
just executing the script directly, instead of calling it with the Python
executable.

--- a/pebble_tool/sdk/emulator.py
+++ b/pebble_tool/sdk/emulator.py
@@ -304,7 +304,6 @@
                                    "layouts.json")

         command = [
-            sys.executable,
             phonesim_bin,
             "--qemu", "localhost:{}".format(self.qemu_port),
             "--port", str(self.pypkjs_port),
